contrib/NeonVM/neonvm.c

```diff
@@ -416,37 +396,38 @@ void exec_LOADB()
 void exec_LOADN(struct tagTExecutor *self)
 {
     self->ip += 1;
-    Cell *addr = top(self->stack)->address; pop(self->stack);
+    Cell *addr = top(self->stack)->address;
     push(self->stack, addr);
+    pop(self->stack);

```

Comment:
You can't pop after pushing (you just popped the thing you pushed in the line before). You have to pop the stack *before* pushing the new value.